src/test/cross-browser/codecept.conf.ts

```diff
@@ -4,13 +4,12 @@ import { config as testConfig } from '../config';
 
 setHeadlessWhen(testConfig.TestHeadlessBrowser);
 
-const url = testConfig.TEST_URL || 'http://localhost:3001';
 let helpers = {};
 let plugins = {};
 if (process.env.SAUCE === 'true') {
   helpers = {
     WebDriver: {
-      url,
+      url: testConfig.TEST_URL,
```

Comment:
should we not have an or condition like above?

Reply:
I've moved it up into the main config so `testConfig.TEST_URL` still doing `process.env.TEST_URL || 'http://localhost:3001'` 👍